osu.Game/Collections/CollectionToggleMenuItem.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Beatmaps;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Collections
+{
+    public class CollectionToggleMenuItem : ToggleMenuItem
+    {
+        public CollectionToggleMenuItem(BeatmapCollection collection, IBeatmapInfo beatmap)
+            : base(collection.Name.Value, MenuItemType.Standard, s =>
+            {
+                if (s)
```

Comment:
rename this to anything but `s`

Reply:
Copy-pasted from previous place (in carousel beatmap class), but do agree with that.